advtraj/integrate/__init__.py

```diff
@@ -95,26 +83,28 @@ def integrate_trajectories(
     else:
         raise NotImplementedError(times)
 
+    ref_time = ds_starting_points.time
+    ds_starting_points = ds_starting_points.assign_coords({"ref_time": ref_time})
```

Comment:
Do we need to assign `ref_time` into a coordinate here? I don't see it being used anywhere else apart from just below when producing the forward and backward by slicing using `ref_time`

Reply:
At present, we're calculating datasets for a single ref_time, but it's very useful to calculate 'families' with different ref_times, in which case, yes we need it to merge datasets together.